### DISCUSSION TITLE
WinFsp 1.x uninstaller for WixBurnBundles

### DISCUSSION THREAD

**Discussion by infeo** [winfsp/winfsp#519] [Show and tell]
Hey, since the WinFsp installer up to version 2.0.0 cannot simply upgrade an already preinstalled version and requires uninstalltion of the preinstalled one, this task needs to made by the installation framework.

For Cryptomator we developed a small WinFsp 1.x EXE uninstaller usable in [Wix Burn bundles](https://wixtoolset.org/docs/tools/burn/), see https://github.com/cryptomator/winfsp-uninstaller. You can give us feedback how it works in your case!

**Comment by billziss-gh** [comment]
@infeo I have been away for some time due to a personal family matter, but I will have a look now that I am back. Thanks!
